types/constants.go

```diff
@@ -48,6 +48,10 @@ var (
 	OakMaxRise              *big.Rat
 	OakMaxDrop              *big.Rat
 	OakHardforkTxnSizeLimit = uint64(64e3) // 64 KB
+
+	FileContractRenterIndex = uint(0)
+	FileContractHostIndex   = uint(1)
+	FileContractVoidIndex   = uint(2)
```

Comment:
makes more sense to define these in the `modules` package with the rest of the negotiation code